media-proxy/src/proxy_context.cc

```diff
@@ -798,8 +832,9 @@ int ProxyContext::RxStart(const mcm_conn_param* request)
         rx_udp_h264_session_context_t* rx_ctx = NULL;
         // mcm_dp_addr remote_addr = request->remote_addr;
         mcm_dp_addr local_addr = request->local_addr;
-        // rx_ctx = mtl_udp_h264_rx_session_create(mDevHandle, &remote_addr, &memif_ops);
-        rx_ctx = mtl_udp_h264_rx_session_create(mDevHandle, &local_addr, &memif_ops);
+        /*udp poll*/
+        //rx_ctx = mtl_udp_h264_rx_session_create(mDevHandle, &remote_addr, &memif_ops);
```

Comment:
this line of code can be removed instead of commented-out.

Reply:
done